src/main/java/org/jspecify/annotations/Nullable.java

```diff
@@ -110,12 +111,12 @@
  * following cases, where they have no defined meaning:
  *
  * <ul>
- *   <li>On a <b>local variable</b> declaration. According to JSpecicy, the nullness of a local
- *       variable itself is not a fixed declarative property of its type. Rather it should be
- *       inferred from the nullness of each expression assigned to the variable, possibly changing
- *       over time. (<a href="https://bit.ly/3ppb8ZC">Why?</a>). Note that this rule applies only to
- *       the <i>root type</i> of the variable; other type components (type arguments, array
- *       component types) are annotatable as usual.
+ *   <li>On a <b>local variable</b> declaration. JSpecify treats the nullness of a local variable as
+ *       being a purely dynamic property, not being a fixed declarative property of its type. (<a
```

Comment:
```suggestion
 *       being a purely dynamic property, not a fixed declarative property of its type. (<a
```

Also, maybe "declared" is simpler than "declarative"? And maybe "static" is better than "fixed", to contrast with "dynamic"?